pkg/protoassert/assert.go

```diff
@@ -0,0 +1,65 @@
+package protoassert
+
+import (
+	"testing"
+
+	"github.com/golang/protobuf/jsonpb"
+	"github.com/golang/protobuf/proto"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/exp/maps"
+)
+
+func Equal(t *testing.T, expected, actual proto.Message, msgAndArgs ...interface{}) bool {
+	t.Helper()
+	e, err := toJson(expected)
+	require.NoError(t, err)
+	a, err := toJson(actual)
+	require.NoError(t, err)
+	return assert.JSONEq(t, e, a, msgAndArgs)
+}
+
+func SlicesEqual[T proto.Message](t *testing.T, expected, actual []T, msgAndArgs ...interface{}) bool {
+	t.Helper()
+	areEqual := assert.Len(t, actual, len(expected), msgAndArgs)
+	for i, e := range expected {
+		a := actual[i]
+		areEqual = areEqual && assert.Equal(t, a, e, msgAndArgs)
```

Comment:
```suggestion
		areEqual = areEqual && Equal(t, a, e, msgAndArgs)
```
I guess the point is to ensure that all messages in the input arrays are identical.